modules/ROOT/pages/apply-configure-json-threat-task.adoc

```diff
@@ -9,17 +9,23 @@ endif::[]
 >s|Policy Name|JSON Threat Protection
 >s|Summary|Protects against malicious JSON in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
 >s|Returned Status Codes|400 - Bad Request
 |===
 
-== Purpose
+JSON requests are susceptible to attacks characterized by unusual inflation of elements and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The JSON Threat Protection policy helps protect your applications from such intrusions.
 
-JSON requests are susceptible to attacks characterized by unusual inflation of elements and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The JSON threat protection policy helps protect your applications from such intrusions.
+If Mule runtime engine (Mule) fails to detect an attack, you must monitor and design your services architecture with layers of protection in addition to these policies.
```

Comment:
"If Mule runtime engine (Mule) fails to detect an attack": This statement seems to imply that there is in place a complex system that comes out of the box (or can be enabled somehow), that monitors for attacks in general, which is not correct. No such system exists today or will exist in the near future. Perhaps you may say “There are other possible attack vectors. Consider designing your services architecture with other layers of protection in addition to this one.”

Reply:
Done! Thanks!